src/main/java/racingcar/domain/RacingCar.java

```diff
@@ -0,0 +1,52 @@
+package racingcar.domain;
+
+import racingcar.domain.strategy.MoveStrategy;
+
+public class RacingCar implements Car {
+
+    private static final int NAME_SIZE = 5;
+
+    private final String name;
+    private int position;
+
+    public RacingCar(String name) {
+        isCorrectNameSize(name);
+        this.name = name;
+        this.position = 0;
+    }
+
+    public RacingCar(String name, int position) {
+        this(name);
+        this.position = position;
```

Comment:
`주 생성자`는 무엇일까요? 🤔
아래와 같이 변경하는 것이 좋을 것 같아요! 😃

```suggestion
    public RacingCar(String name) {
        this(name, 0);
    }

    public RacingCar(String name, int position) {
        isCorrectNameSize(name);
        this.name = name;
        this.position = position;
```

[참고 자료](https://skyblue300a.tistory.com/6)